src/Plugin/Field/FieldWidget/ProjectGenusWidget.php

```diff
@@ -0,0 +1,301 @@
+<?php
+
+namespace Drupal\trpcultivate\Plugin\Field\FieldWidget;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\StringTranslation\TranslatableMarkup;
+use Drupal\tripal\TripalField\Attribute\TripalFieldWidget;
+use Drupal\tripal_chado\TripalField\ChadoWidgetBase;
+use Drupal\tripal_chado\Controller\ChadoOrganismFormElementController;
+
+/**
+ * Plugin implementation of the 'project_genus_widget' field widget.
+ */
+#[TripalFieldWidget(
+  id: 'project_genus_widget',
+  label: new TranslatableMarkup('Project Genus Select List'),
+  description: new TranslatableMarkup('Provides a select list of organisms in this site.'),
+  field_types: [
+    'project_genus',
+  ],
+)]
+class ProjectGenusWidget extends ChadoWidgetBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
+
+    // Get the field settings.
+    $field_definition = $items[$delta]->getFieldDefinition();
+    $field_name = $field_definition->get('field_name');
+
+    $options = [];
+    // Set some defaults to keep each of the fields simpler.
+    $options['select_limit'] = $this->getSelectLimit($options['select_limit'] ?? NULL);
+    $options['match_operator'] ??= $this->getSetting('match_operator') ?? 'CONTAINS';
+    $options['match_limit'] ??= $this->getSetting('match_limit') ?? 10;
+    $options['size'] ??= $this->getSetting('size');
+    $options['placeholder'] ??= $this->getSetting('placeholder');
+
+    // Get the select options for the organism select list.
+    $select_options = ChadoOrganismFormElementController::getSelectOptions($options);
+
+    // Current Item Values:
+    $item_vals = $items[$delta]->getValue();
+    $record_id = $item_vals['record_id'] ?? 0;
+
+    // Determine the current organism_id based on the scientific name value.
+    $organism_id = 0;
+    if (array_key_exists('sciname_value', $item_vals) and $item_vals['sciname_value'] != '') {
+      $sciname_string = ChadoOrganismFormElementController::getQuery($item_vals['sciname_value'], [])->execute()->fetchAll()[0]->organism;
+      foreach ($select_options as $id => $sciname) {
+        if ($sciname == $sciname_string) {
+          $organism_id = $id ?? 0;
+        }
+      }
+    }
+
+    // ID space manager to get the terms later.
+    $idSpace_manager = \Drupal::service('tripal.collection_plugin_manager.idspace');
+
+    $elements = [];
+    $elements['record_id'] = [
+      '#type' => 'value',
+      '#value' => $record_id,
+    ];
+    $elements['field_name'] = [
+      '#type' => 'value',
+      '#value' => $field_name,
+    ];
+
+    // GENUS.
+    $genus_prop_id = $item_vals['genus_prop_id'] ?? 0;
+    $genus_prop_fkey = $item_vals['genus_prop_fkey'] ?? 0;
+    $genus_value = $item_vals['genus_value'] ?? '';
+    // -- get the term.
+    $idSpace_taxrank = $idSpace_manager->loadCollection('TAXRANK');
```

Comment:
See if $this->getFieldSetting('termIdSpace'); and $this->getFieldSetting('termAccession'); work here for the hard coded field values.